src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/upgrade-v2.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: Upgrade from v2
+tags:
+    - Integrations
+    - Kubernetes integration
+    - Changelog
+metaDescription: Upgrade from v2
+redirects:
+    - /docs/kubernetes-pixie/kubernetes-integration/get-started/changes-since-v3
+freshnessValidatedDate: 2024-07-22
+---
+
+In order to upgrade the Kubernetes integration from version 2 (included in the [`nri-bundle`](https://github.com/newrelic/helm-charts/tree/master/charts/nri-bundle) chart versions 3.x), we strongly encourage you to create a `values-newrelic.yaml` file with your desired <InlinePopover type="licenseKey" /> and configuration. If you had previously installed our chart from the CLI directly, for example using a command like the following:
+
+```shell
+helm install newrelic/nri-bundle \
+  --set global.licenseKey=<New Relic License key> \
+  --set global.cluster=<Cluster name> \
+  --set infrastructure.enabled=true \
+  --set prometheus.enabled=true \
+  --set webhook.enabled=true \
+  --set ksm.enabled=true \
+  --set kubeEvents.enabled=true \
+  --set logging.enabled=true
+```
+
+You can take the provided `--set` arguments and put them in a yaml file like the following:
+
+```yaml
+# values-newrelic.yaml
+global:
+  licenseKey: <New Relic License key>
+  cluster: <Cluster name>
+infrastructure:
+  enabled: true
+prometheus:
+  enabled: true
+webhook:
+  enabled: true
+ksm:
+  enabled: true
+kubeEvents:
+  enabled: true
+logging:
+  enabled: true
+```
+
+After doing this, and adapting any other setting you might have changed according to the [migration guide](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3/#migration-guide), you can upgrade your `nri-bundle` by running this command:
```

Comment:
```suggestion
After doing this, and adapting any other setting you might have changed according to the [migration guide](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3/#migration-guide), upgrade your `nri-bundle` by running this command:
```